src/core/tests/Test_CoreMain.py

```diff
@@ -243,7 +248,8 @@ def test_invalid_maintenance_run_id(self):
         self.assertTrue(substatus_file_data[2]["name"] == Constants.PATCH_METADATA_FOR_HEALTHSTORE)
         self.assertTrue(substatus_file_data[2]["status"].lower() == Constants.STATUS_SUCCESS.lower())
         substatus_file_data_patch_metadata_summary = json.loads(substatus_file_data[2]["formattedMessage"]["message"])
-        self.assertEqual(substatus_file_data_patch_metadata_summary["patchVersion"], maintenance_run_id)
+        self.assertEqual(substatus_file_data_patch_metadata_summary["patchVersion"], health_store_id)
+        self.assertEqual(runtime.execution_config.max_patch_publish_date, "20240401T000000Z")
```

Comment:
We can also add a check that runtime.package_manager.max_patch_publish_date is str()

Reply:
Isn't that implicitly covered in the equality check?

Reply:
This was for max_patch_publish_date  within package_manager, which will be str() since this is Zypper. Not as important since it is covered in later tests. Resolving this comment